--- Makefile.PL~	Mon Sep 14 12:37:25 1998
+++ Makefile.PL	Fri Oct  8 22:53:03 1999
@@ -1,7 +1,7 @@
 
 use ExtUtils::MakeMaker;
 
-my $dir = '.:/packages/share/tex/lib/fonts';
+my $dir = '/usr/share/texmf/fonts';
 
 eval 'use Font::TFM';
 if ($@ eq '' and defined $Font::TFM::TEXFONTSDIR)
@@ -12,9 +12,6 @@
 EOF
 	$dir = $Font::TFM::TEXFONTSDIR;
 	}
-
-$dir = prompt "Where should I search for tfm files (value of \$Font::TFM::TEXFONTSDIR)\n", $dir;
-
 
 sub MY::processPL
 	{
